src/tools/rados/rados.cc

```diff
@@ -4228,13 +4283,26 @@ int main(int argc, const char **argv)
     } else if (ceph_argparse_witharg(args, i, &val, "-o", "--output", (char*)NULL)) {
       opts["output"] = val;
     } else if (ceph_argparse_flag(args, i, "--write-omap", (char*)NULL)) {
-      opts["write-dest-omap"] = "true";
+      // write- prefixed dests are legacy
+      opts["dest-omap"] = "true";
     } else if (ceph_argparse_flag(args, i, "--write-object", (char*)NULL)) {
-      opts["write-dest-obj"] = "true";
+      opts["dest-obj"] = "true";
     } else if (ceph_argparse_flag(args, i, "--write-xattr", (char*)NULL)) {
-      opts["write-dest-xattr"] = "true";
+      opts["dest-xattr"] = "true";
+    } else if (ceph_argparse_flag(args, i, "--read-omap", (char*)NULL)) {
```

Comment:
With the current proposal one can successfully invoke something like:
rados bench read --write-omap
Which is a bit confusing.

Wouldn't it be better to tag "--write-*" options as deprecated and introduce "--object"/"--omap"/"--xattr" ones or something?